nixos/modules/installer/tools/nixos-install.sh

```diff
@@ -245,6 +245,7 @@ if [[ -z $noBootLoader ]]; then
     ln -sfn /proc/mounts "$mountPoint"/etc/mtab
     export mountPoint
     NIXOS_INSTALL_BOOTLOADER=1 nixos-enter --root "$mountPoint" -c "$(cat <<'EOF'
+      set -e
```

Comment:
This kind of scares me just for being a meaningful change to an important installation tool rather than something test‐oriented, would you object to splitting it out into another PR?